### ISSUE TITLE
ERR: Not able to publish new thread using the API v1.3.1

### ISSUE THREAD

**Issue by sudarshanmg** [junhoyeo/threads-api#115]
## Problem:
- I tried using the latest version `1.3.1` of `threads-api`
- Publishing new thread using the `publish()` method fails using the code below:

  ![image](https://github.com/junhoyeo/threads-api/assets/86506425/4a81eca8-c9eb-43cc-beb8-5a24d76312ff)

- Instead, it returns an error:

  ![image](https://github.com/junhoyeo/threads-api/assets/86506425/14b376fb-9249-4256-8730-9226c199f430)

- All the other methods works perfectly fine.

**Comment by yekayee** [comment]
same problem

**Comment by Aerglonus** [comment]
Try to get the token with  and pass it manually
```
const token = await threadsAPI.getToken()
```

```
  const threadsAPI = new ThreadsAPI({
    token: `${token}`,
    username: '${username}', // Your username
    password: `${password}`, // Your password
  });
```

**Comment by sudarshanmg** [comment]
In your code,
You cannot access `threadsAPI` before initialization right?
You are creating a new object named `threadsAPI` after you are trying to access it to get token.

**Comment by yekayee** [comment]
any example?

**Comment by sudarshanmg** [comment]
Sorry I was replying to @Aerglonus 's code

**Comment by LilaRest** [comment]
Same here, it seems that the problem comes from authentication.
After examining the output of `const { data } = await axios<string>(LOGIN_URL, requestConfig);` in `getToken()` method I found error messages:
- `Unable to log in`
- `An unexpected error occurred. Please try logging in again.`

**Comment by Aerglonus** [comment]
> In your code, You cannot access `threadsAPI` before initialization right? You are creating a new object named `threadsAPI` after you are trying to access it to get token.

> can u give me example? sorry im newbie

```
const main = async () => {
  const threadsAPI = new ThreadsAPI({
    username: 'username', // Your username
    password: 'password', // Your password
  });
  const token = await threadsAPI.getToken()
  console.log(token)
}
```
This is how I got the token and I was able to post again but apparently the `getToken` its failing #113  (with the token I got I'm still able to post )

**Comment by yekayee** [comment]
Username and password are required

**Comment by yekayee** [comment]
> 

![image](https://github.com/junhoyeo/threads-api/assets/17975832/12f3cb95-0c79-4ff6-84f6-a9be68d70ebb)

**Comment by francistogram** [comment]
I think we’re all having the same issue and my guess is that Threads changed how their auth flow worked so this code broke because it expected the auth token response in a certain format

https://github.com/junhoyeo/threads-api/issues/113

**Comment by junhoyeo** [comment]
Hey all, I released [v1.4.0](https://github.com/junhoyeo/threads-api/releases/tag/v1.4.0) with encrypted passwords. Please try out! 🚀
